@xen-orchestra/web/src/layouts/AppLayout.vue

```diff
@@ -19,13 +19,14 @@
       <AccountMenu />
     </template>
     <template #sidebar-header>
-      <SidebarSearch v-model="filter" :is-searching />
+      <SidebarSearch v-model="filter" />
     </template>
     <template #sidebar-content>
       <VtsTreeList v-if="!isReady">
         <VtsTreeLoadingItem v-for="i in 5" :key="i" icon="fa:city" />
       </VtsTreeList>
       <NoResults v-else-if="sites.length === 0" />
+      <VtsStateHero v-else-if="isSearching" format="card" busy size="medium" class="loader" />
       <SiteTreeList v-else :branches="sites" />
```

Comment:
```suggestion
<VtsStateHero v-else-if="isSearching" format="card" busy size="medium" class="loader" />
<NoResults v-else-if="sites.length === 0" />
<SiteTreeList v-else :branches="sites" />
```

You must place `VtsStateHero` in first place because otherwise if you search for a term that does not exist and you delete the last character the loader does not appear